src/test/resources/Features/OCD-2533-eCQMNumberandVersionUpdate.feature

```diff
@@ -0,0 +1,29 @@
+@Regression
+Feature: OCD-2533 -  Update eCQM numbers and versions for 2019 reporting bundle
+
+Scenario Outline: Verify newly added CQMs in listing details
+    Given I am on listing details page of listing with ID "<DB_ID>"
+    When I look at CQM details
+    Then I see newly added CQMs "<CMSID_title>"
```

Comment:
Did the ticket require adding new CQMs to a particular set of listings? If not, how can you be sure the listings in your scenarios will have these CQMs? Or are you just checking that they're in the list of "available" ones, without checking that the Listing attests to either?

Reply:
The latter. I am opening list of all criteria and CQM to just view that newly added values show on listing details (not attested to).